doc/api/process.md

```diff
@@ -483,6 +483,16 @@ $ bash -c 'exec -a customArgv0 ./node'
 'customArgv0'
 ```
 
+## process.bits
```

Comment:
consider putting this in the config builtin: https://github.com/nodejs/node/blob/master/src/node_config.cc#L21-L24